.github/workflows/test.yaml

```diff
@@ -82,18 +80,16 @@ jobs:
           ~/.vvm
         key: compiler-cache
 
-    - name: Cache compiler installations
+    - name: Cache Brownie Build Folders
       uses: actions/cache@v2
       with:
         path: |
-          ~/.solcx
-          ~/.vvm
-        key: ${{ runner.os }}-compiler-cache
+          ~/.brownie
+          ./build
```

Comment:
with this, ci would compile only the diff.

Reply:
Yep!